src/plugins/validation.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Validator that validates input meets a minimum length
+ * Returns the error message otherwise.
+ *
+ * @param {number} length The minimum length allowed for the field's value
+ * @param {string} message
+ * @returns {function(*): (boolean|string)}
+ */
+const minLength = (
+  length: number,
+  message?: string
+): ((v: any) => string | true | undefined) => {
+  message = message || `Min ${length} characters allowed.`;

```

Comment:
- [ ] Let's use not use `any` if we can help it.  Replace `v:any` with `v: string | number | boolean` here and throughout the `validation.ts` and I think that should do it.